admin/Makefile

```diff
@@ -18,17 +22,17 @@ check: flake8
 
 .PHONY: flake8
 flake8: ${BUILD_DIR}/requirements-dev.timestamp
-	${VENV}/bin/flake8 c2cgeoportal_admin acceptance_tests
+	${VENV}/bin/flake8 c2cgeoportal_admin acceptance_tests *.py
 
 .PHONY: init_db
 init_db:
 	#Configure the database:
-	${VENV}/bin/initialize_db_main development.ini
+	${VENV}/bin/initialize_db_main production.ini
 
 .PHONY: serve
 serve: ${BUILD_DIR}/requirements.timestamp
 	#Run your project.
-	${VENV}/bin/pserve development.ini
+	${VENV}/bin/pserve production.ini
```

Comment:
Why pserve use the production environment?

Reply:
because it seemed to me that the c2cwsgiutils project we're based on only uses a single ini file named production.ini but I might be wrong...

Reply:
For me it don't mean thet we should remove the development.ini file ...